opensearch-operator/pkg/builders/cluster.go

```diff
@@ -179,6 +179,21 @@ func NewSTSForNodePool(
 
 	image := helpers.ResolveImage(cr, &node)
 
+	var mainCommand []string
+	var com string
+	com = "yes | ./bin/opensearch-plugin install"
```

Comment:
```suggestion
	com = "./bin/opensearch-plugin install -b"
```

you should use the batch mode. then you also don't need the `yes` crutch.

i just noticed that this isn't documented yet for OpenSearch and have raised https://github.com/opensearch-project/documentation-website/issues/731 for this. see [the ES docs](https://www.elastic.co/guide/en/elasticsearch/plugins/7.10/_other_command_line_parameters.html#_batch_mode) instead for now.